### PR TITLE
Add some documentation

### PR DESCRIPTION
**Pull request by chrislo**
Add some documentation as per #103 and #104

### CODE REVIEW

File: docs/costing.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Estimated cost of AWS storage
+
+To store all of the [current assets](existing_assets.md) in Asset Manager and Whitehall would require ~670 GB. S3 storage is currently priced at [$0.023/GB/month on S3](https://aws.amazon.com/s3/pricing/) which equates to ~$15/month.
+
+Amazon also offers an [Elastic File System (EFS)](https://aws.amazon.com/efs/) in the [Ireland and Frankfurt](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) availability zones. It appears to have the advantage over EBS in that the volumes scale automatically with the data that is added. As it can be mounted as a file-system to an EC2 instance it potentially offers an alternative for Asset Manager that would require smaller changes to the existing AM codebase (in that the mounted EFS system would appear to the asset manager application as a file system like the current NFS model).
+
+EFS is more expensive than S3, [currently priced](https://aws.amazon.com/efs/pricing/) at $0.33/GB/month or ~$221/month for ~670GB.
```
Comment (floehopper) [inline docs/costing.md]: Might it be worth noting that there might also be transfer/bandwidth costs...?

File: docs/existing_assets.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+@integration-asset-master-1:/mnt/uploads$ du -h --max-depth=1
+35G ./asset-manager
+du: cannot read directory ‘./whitehall/clean/system/uploads/attachment_data/file/462737’: Permission denied
+629G    ./whitehall
+7.6M    ./publisher
+14G ./support-api
+du: cannot read directory ‘./lost+found’: Permission denied
+16K ./lost+found
+677G    .
+```
```
Comment (floehopper) [inline docs/existing_assets.md]: I'm not sure it's worth inclusing the "Permission denied" lines...

File: docs/existing_assets.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+The asset manager application stores a record in MongoDB for each asset. On integration the number of records is
+
+``` bash
+@integration-backend-1:/var/apps/asset-manager$ sudo su - deploy
+deploy@integration-backend-1:~$ cd /var/apps/asset-manager
+deploy@integration-backend-1:/var/apps/asset-manager$ govuk_setenv asset-manager bundle exec rails c
+Loading production environment (Rails 4.2.7.1)
+irb(main):001:0> Asset
+=> Asset
```
Comment (floehopper) [inline docs/existing_assets.md]: Minor: The preceding two lines could be removed.

**Comment by chrislo** [comment]
@floehopper I made those changes, rebased and force-pushed. Could you re-approve please?

**Pull request merged**